controllers/statusreport/statusreport_controller.go

```diff
@@ -101,7 +108,10 @@ func SetupController(manager ctrl.Manager, log *logr.Logger) error {
 func setupControllerWithManager(manager ctrl.Manager, controller *Reconciler) error {
 	return ctrl.NewControllerManagedBy(manager).
 		For(&applicationapiv1alpha1.Snapshot{}).
-		WithEventFilter(predicate.Or(
-			gitops.SnapshotTestAnnotationChangePredicate())).
+		WithEventFilter(
+			predicate.And(
+				toolkitpredicates.IgnoreBackups{},
```

Comment:
Technically we don't need this predicate here as we don't react to creation in other predicates. But better safe than sorry, LMKWDYT

Reply:
I agree, it's better to have it here in case something changes in the future.